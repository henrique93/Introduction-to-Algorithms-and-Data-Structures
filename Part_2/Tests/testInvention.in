10
send @Wallace #Cheese I do like a bit of gorgonzola.
send @Wallace #Invention It's time we tried my latest invention, the Mind Manipulation-omatic. It extracts unwanted thoughts and desires.
send @Wallace #Invention I haven't tested it yet, but it should be perfectly safe. Just a bit of harmless brain alteration, that's all.
list #Invention
close @Wallace #Invention
send @Wendolene #Cheese Oh no, not cheese. Sorry. Brings me out in a rash. Can't stand the stuff.
send @Wallace #Cheese I'm just crackers about cheese!
list #Invention
list #Cheese
quit
