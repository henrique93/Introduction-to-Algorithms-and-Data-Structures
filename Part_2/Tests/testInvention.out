@Wallace #Invention : It's time we tried my latest invention, the Mind Manipulation-omatic. It extracts unwanted thoughts and desires.
@Wallace #Invention : I haven't tested it yet, but it should be perfectly safe. Just a bit of harmless brain alteration, that's all.
NULL
@Wallace #Cheese : I do like a bit of gorgonzola.
@Wendolene #Cheese : Oh no, not cheese. Sorry. Brings me out in a rash. Can't stand the stuff.
@Wallace #Cheese : I'm just crackers about cheese!
